docs/manifest.json

```diff
@@ -1212,7 +1212,20 @@
                 ]
               ]
             },
-            { "title": "Core 1 Documentation", "href": "/docs/core-1", "icon": "link" }
+            { "title": "Core 1 Documentation", "href": "/docs/core-1", "icon": "link" },
+            {
+              "title": "Dashboard",
```

Comment:
💬 Should we create another section specific to Dashboard features? I haven't found one with similar purposes.